components/uniset/src/error.rs

```diff
@@ -0,0 +1,46 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/master/LICENSE ).
+use std::fmt;
+use std::path::{Path, PathBuf};
+
+#[derive(Debug)]
+pub enum Error {
+    Io(std::io::Error, Option<PathBuf>),
```

Comment:
The uniset crate doesn't perform any I/O, so you shouldn't need an I/O error.

Reply:
Done.

Reply:
Why didn't you delete the I/O error?

Reply:
Done.